src/index.js

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
```

Comment:
The changes in this files have been made by eslint so it just include cosmetics.